packages/astro/src/core/build/generate.ts

```diff
@@ -150,7 +170,7 @@ export async function generatePages(opts: StaticBuildOptions, internals: BuildIn
 	}
 
 	const verb = ssr ? 'prerendering' : 'generating';
-	logger.info(null, `\n${bgGreen(black(` ${verb} static routes `))}`);
+	logger.info('SKIP_FORMAT', `\n${bgGreen(black(` ${verb} static routes `))}`);
```

Comment:
Using a label to "skip" the formatting seems fine to me, but not in this way:
- it's not documented anywhere
- it gives a precedent to add more "known labels" to stuff that isn't related to the label, but to the log itself

I would prefer to have this as an alternative:
1. `logger.info("label", "msg", false)`, where the `false` is to say that it's an unformatted log
2. `logger.unformatted_info("label", "msg")`|

I prefer `2.`. This gives us a way to document what's an "unformatted log"

Reply:
I actually agree with you that a more formal API would be great here. I was just intentionally trying to keep any API changes out of scope for this PR to keep focus as much as possible.

Do you have any objection to keeping `SKIP_FORMAT` in this PR, and then me making a separate PR with a more formal API as a fast-follow?

I'm happy to do either! But I could see us going back-and-forth on that API and this PR is already large enough & far-reaching enough as it is, I wanted to separate the two.

Reply:
I'm happy with doing a follow up PR!

Reply:
Update: I made some small changes here to type this in TS, to keep our `label` values fairly standardized, prevent typos, etc. 

This actually feels like a more formal API now, so possibly worth another look if this can be the final API. Otherwise, still happy to do that follow-up PR.

Reply:
I definitely like this better than everything being undocumented! I personally dislike our label system, so I'd be happy with a follow-up to reformat, remove, or refactor the API for them.

The current API feels odd because the very first argument is optional (and we're now using a special string to opt-out instead of `undefined` or `null`). It would be better if the logger supported an options bag with a `label` as the last argument.